package.json

```diff
@@ -49,11 +49,12 @@
   "scripts": {
     "docs:build": "gitbook install && gitbook build",
     "docs:watch": "gitbook install && gitbook serve",
-    "prepublish": "nwb build",
+    "prepublish": "nwb build && npm run types",
     "release": "semantic-release pre && npm publish && semantic-release post",
     "test": "node umd/skatejs.js && semistandard && npm run test:ts && nwb test -- --ci",
     "test:ts": "tsfmt -r && tsc -p ./",
-    "test:watch": "nwb test --server"
+    "test:watch": "nwb test --server",
+    "types": "../../treshugart/get-typed/bin/get-typed.js src/index.js --dest umd/skatejs"
```

Comment:
where is definition of this weird `_Class` type? https://gist.github.com/treshugart/9bbbba9d524dd75eb2ccf6b4b21df706#file-skatejs-d-ts-L17